File: src/C#/BehavioralPatterns/Command/FastFoodRestaurant/Commands/AddMenuItemCommand.cs
using FastFoodRestaurant.Models;
using System;
using System.Collections.Generic;
using System.Text;

namespace FastFoodRestaurant.Commands
{
    public class AddMenuItemCommand : OrderCommand
    {
        public override void Execute(List<MenuItem> fastFoodOrderMenuItems, MenuItem menuItem)
        {
            fastFoodOrderMenuItems.Add(menuItem);
        }
    }
}

File: src/PHP/BehavioralPatterns/Command/FastFoodRestaurant/Commands/AddMenuItemCommand.php
<?php

namespace BehavioralPatterns\Command\FastFoodRestaurant\Commands;

use BehavioralPatterns\Command\FastFoodRestaurant\Models\MenuItem;

class AddMenuItemCommand extends OrderCommand {
    public function execute(array &$fastFoodOrderMenuItems, MenuItem $menuItem): void
    {
        $fastFoodOrderMenuItems[] = $menuItem;
    }
}
